### ISSUE TITLE
Push.feedback prints exceptions to stderr.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ntrees/javapns#90]
```
What is the expected output? What do you see instead?

If an exception occurs in Push.feedback() it's written to stderr. Please 
forward it to the caller.

What version of the product are you using? On what operating system?

javapns 2.1, MacOS X 10.7.2
```

Original issue reported on code.google.com by `thomas...@googlemail.com` on 7 Dec 2011 at 1:42
- Merged into: #86

**Comment by GoogleCodeExporter** [comment]
Original comment by `sype...@gmail.com` on 7 Dec 2011 at 3:08
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
Note: fix has been in the trunk since Nov. 22nd (r336), and trunk version has 
been published as 2.1.1 today (you can get it from the Downloads page right 
now).
```

Original comment by `sype...@gmail.com` on 7 Dec 2011 at 4:21
